tests/st/no_orchestrator/test_mainline_single_host.py

```diff
@@ -17,6 +17,39 @@
 from tests.st.utils.docker_host import DockerHost
 
 class TestNoOrchestratorSingleHost(TestBase):
+    def test_single_host_auto_ipv4(self):
```

Comment:
Hmm, actually this needs some docs or something to clarify.  There are two bits of auto detection going on here: (1) auto-detecting in the test framework so that we pass the IPv4 address through into calicoctl, and the other is calicoctl auto detecting the IPv4 address when it is not supplied.   It's not immediately obvious which this is without sifting through the code.